backend/lib/tasks/transifex.rake

```diff
@@ -2,7 +2,7 @@ namespace :transifex do
   desc "Pulling transifex translations and post processing"
   task :pull do
     puts "transifex_pull: Running TX client to pull translations"
-    IO.popen("tx pull -f -a") { |p| p.each { |line| puts line } }
+    IO.popen("/usr/tx-cli/tx pull -f -a") { |p| p.each { |line| puts line } }
```

Comment:
I'm quite unsure about this change.

Reply:
this will create a system-wide dependency, as well as some evil breaks (if, for any reason, anyone is running this on windows, it now stopped working)